src/server/pfs/s3/object.go

```diff
@@ -70,6 +85,23 @@ func (c *controller) GetObject(r *http.Request, bucketName, file, version string
 	return &result, nil
 }
 
+func (lfs *localFS) getObject(pc *client.APIClient, bucket *Bucket, path, version string) (*s2.GetObjectResult, error) {
+	fullPath := filepath.Join(bucket.Path, path)
+	info, err := os.Stat(fullPath)
+	if err != nil {
+		return nil, err
+	}
+
+	file, err := os.Open(fullPath)
+	if err != nil {
+		return nil, err
+	}
+	return &s2.GetObjectResult{
+		ModTime: info.ModTime(),
+		Content: file,
```

Comment:
Do we still want to include the other fields here such as ETag, Version, DeleteMarker?

Reply:
`DeleteMarker` should be false implicitly, but I can set it as above. `ETag` is a bit more complicated. The S3 spec lists it as optional, and I'm not sure it makes sense to figure out how to efficiently provide file hashes (presumably caching them somehow) until we're sure integrations need them. I commented out a line in the tests that expects them, but it's definitely possible we'll need them later.